packages/main/src/webComponents/List/List.mdx

```diff
@@ -0,0 +1,54 @@
+import { ArgTypesWithNote, ControlsWithNote, DocsHeader, Footer } from '@sb/components';
+import SubcomponentsSection from '@sb/docs/SubcomponentsSection.md?raw';
+import { Canvas, Description, Markdown, Meta } from '@storybook/blocks';
+import '@ui5/webcomponents-icons/dist/employee.js';
+import { CustomListItem } from '../CustomListItem';
+import CustomListItemDescription from '../CustomListItem/CustomListItemDescription.md?raw';
+import { GroupHeaderListItem } from '../GroupHeaderListItem';
+import GroupHeaderListItemDescription from '../GroupHeaderListItem/GroupHeaderListItemDescription.md?raw';
+import { StandardListItem } from '../StandardListItem';
+import StandardListItemDescription from '../StandardListItem/StandardListItemDescription.md?raw';
+import * as ComponentStories from './List.stories';
+import ListDescription from './ListDescription.md?raw';
+
+<Meta of={ComponentStories} />
+
+<DocsHeader />
+
+<br />
+
+## Example
+
+<Canvas of={ComponentStories.Default} />
+
+## Properties
+
+<ControlsWithNote of={ComponentStories.Default} />
+
+<Markdown>{ListDescription}</Markdown>
+
+<br />
+
+<Markdown>{SubcomponentsSection}</Markdown>
+
+## StandardListItem
+
+<Description of={StandardListItem} />
```

Comment:
The description is already included in the imported markdown description below. Either we have do adjust the markdown description here, or show the whole description (usage, parts section, etc.) on top of the args table.

The same goes for all web-component sub-components.

Reply:
updated.